internal/generator/config/curve.go

```diff
@@ -22,6 +23,29 @@ type Curve struct {
 	FpInfo, FrInfo Field
 	G1             Point
 	G2             Point
+
+	HashE1 *HashSuite
+}
+
+type Isogeny struct {
+
+	//Isogeny to original curve
+	XMap RationalPolynomial
+	YMap RationalPolynomial // The y map is also evaluated on x. The result is multiplied by y.
+}
+
+type RationalPolynomial struct {
+	NumHex []string
```

Comment:
naming wise, I would remove the "Hex" suffix. just a comment that says coefficients are stored in base16 strings is OK.